.eslintrc.cjs

```diff
@@ -69,5 +69,11 @@ module.exports = {
 
 		// These on-by-default rules don't work well for this repo and we like them off.
 		"no-inner-declarations": "off",
+
+		// Formatting rule - does not interfere with Prettier
```

Comment:
Heh, using the terminology I prefer, this rule is both a 💩 _formatting_ rule and a 💅 _stylistic_ rule. The terms are not super standard in linter docs at the moment...

* https://github.com/typescript-eslint/typescript-eslint/issues/6495
* https://github.com/eslint/eslint.org/issues/419

How about maybe...

```suggestion
		// Stylistic concerns that don't interfere with Prettier
```